.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 15510
```

Comment:
Please ignore this config file.

Reply:
Sorry, I type the wrong name in gitignore, I  will delete this config file.